src/store/auth.js

```diff
@@ -39,8 +41,15 @@ export default {
     socket_updateUserData(/* state */) {
       console.log('socket_updateUserData');
     },
-    socket_login(/* state */) {
-      console.log('socket_login');
+    socket_login(state, token) {
+      try {
+        LocalStorage.set('token', token.value);
+        state.user = jwt.decode(token.value, { complete: true }).payload;
+        Notify.create('Logged in successfully');
+      } catch (e) {
+        Notify.create('Failed to login');
```

Comment:
you should test `isSuccessful` and if it's not, notify with the server's message
```suggestion
        Notify.create(token.value);
```

also, I recommend changing `token` to `loginResult` to match the object's name on the backend